### DISCUSSION TITLE
My account has been flagged... <title>

### DISCUSSION THREAD

**Discussion by sercanserdar** [community/community#44072] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My account was marked, and I didn't break any rules.

I opened a ticket to support, but it has not been responded for 2 weeks. 

Can you please raise this topic for the help guys?

**Comment by sercanserdar** [comment, marked as answer]
I talked to support and they fixed the problem thanks.

**Reply by sercanserdar** [reply]
> Hello 👋How are things going on with you

Thank you, it's going very well. And Yours ?

**Reply by rtaxoficial1** [reply]
my account has also been flagged

**Reply by biyangdo** [reply]
I have a same issue. Thx for the review

**Reply by draganali** [reply]
how did you talked with support? do you have email or phone number to share?

**Reply by reynalmoana** [reply]
Same here: I open a ticket to support team, two weeks later, still no answer...

**Reply by hamidmehdizadeh** [reply]
I have the same problem. I'm flagged without doing anything wrong. I have sent a ticket but no answer! Any idea what should I do?

**Reply by D3troit98** [reply]
I've opened a ticket for a week, and still no answer

**Reply by gooddavvy** [reply]
Just contact GitHub support @draganali

**Reply by wisitlongsida1999** [reply]
I have the same issue. I have flagged without break the any rules. Now, I have sent a ticket to GitHub but no any answer back to me. Any one has any solution ?

**Reply by devtmj1123** [reply]
I also have a same problem

**Reply by hugogontijomachado** [reply]
I also have the same issue. I got flagged but I'm not a spammer. This has been hurting me a lot because I couldn't make any more commits, I can't clone my repositories, I can't use copilot and any other tool. 
I sent a ticket to support a little while ago and I'm waiting for a solution....

**Reply by Umi007** [reply]
I also have the same issue

**Reply by mhamidjamilraza** [reply]
Support is even not responding,
I have contacted, my account was not new, it was created in 2019, and has more then 900 commits.
how bad they are at resolving this issue, i had send them mail before month from the main account, but they don't response i send mail again after 1 week no response, after that i send mail from this account not linked with the flagged account they reply to this mail that i should send mail from the Gmail account which was linked with the flagged account, I mean what the hell is this behavior, @github is not responding on associated mail (for approx 1 month and till now) and replayed on other mail in a day.

**Reply by sliyarli** [reply]
> Support is even not responding, I have contacted, my account was not new, it was created in 2019, and has more then 900 commits. how bad they are at resolving this issue, i had send them mail before month from the main account, but they don't response i send mail again after 1 week no response, after that i send mail from this account not linked with the flagged account they reply to this mail that i should send mail from the Gmail account which was linked with the flagged account, I mean what the hell is this behavior, @github is not responding on associated mail (for approx 1 month and till now) and replayed on other mail in a day.

did you fix this issue? mine flagged too and it's been a week since i sent them a ticket

**Reply by ghost** [reply]
Hello @sliyarli, have you received your account back? I have contacted the support, but they haven't respond to my ticket, your message was the latest in the discussion how long did it took you to get your account back?

**Reply by ahmedembeddedxx** [reply]
@sliyarli when did you got your account back? I have submitted a ticket 3 weeks ago, yet I haven't received a response. How long did it took you to get your response?

**Reply by warsecurity** [reply]
Like I think the support is like ... if your account get flagged or suspended . No recovery . Just open a new account bro

**Comment by boech64** [comment]
My account has also been flagged, and I didn't break any rules.

Can you please raise this topic for the help guys?

**Reply by Dmambo** [reply]
my account was flagged also open ticket and haven't gotten any response from the support team

**Reply by Umi007** [reply]
Same

**Reply by mhamidjamilraza** [reply]
same

**Reply by paulospx** [reply]
same

**Reply by HollmanRiveroYahoo** [reply]
same

**Comment by Abhi96k** [comment]
My account has also been flagged, and I didn't break any rules.

Can you please raise this topic for the help guys?

**Reply by Antitrack** [reply]
My account also has been flagged from its creation immidately. This is my first post. What stupid security mechanisms are at work here?

**Reply by jobrien12345** [reply]
I have the same problem.  My account has been flagged and I don't know why.  So my posts are not public now.  How do I contact Github Support?

**Comment by Nikhops2** [comment]
My Account has also been flagged. Could somebody help me?

**Reply by mhamidjamilraza** [reply]
i guess github is dead.

**Comment by employeemanagementcse4a** [comment]
My account was also flagged! I'm looking for solution to resolve this issue. Anybody can help?

**Comment by ahmetarsiv** [comment]


**Comment by ghost** [comment]
MY account has been flagged. I'm new

**Comment by GlennyPennie** [comment]
My account has also been flagged & I don't know why......

**Comment by riteshara1991** [comment]
I am having the same issue. Please support.

**Comment by tatarstunt** [comment]
[My account has been flagged. I didn't break any rules

**Comment by steelcurtn87** [comment]
my account is flagged as well and I just signed in. I don't even know what I am doing yet.

**Comment by steelcurtn87** [comment]
Please help

**Reply by cdxgh** [reply]
If your account has been flagged on a platform or website, it's important to follow the platform's guidelines and policies to address the issue. Here are some general steps you can take:

1. **Review the Reason for Flagging:**
   - Check if there is any specific reason provided for why your account has been flagged. Look for notifications or messages that might explain the situation.

2. **Read the Platform's Terms of Service:**
   - Familiarize yourself with the terms of service or community guidelines of the platform. This will help you understand what actions may have led to the flagging of your account.

3. **Contact Support:**
   - If you believe your account was flagged in error or you are unsure why it was flagged, reach out to the platform's customer support or help center. They can provide clarification and assistance in resolving the issue.

4. **Comply with Platform Policies:**
   - If the flagging is related to a policy violation, take steps to correct the issue. This may involve deleting specific content, adjusting your behavior, or making necessary changes to your account.

5. **Appeal Process:**
   - Some platforms have an appeal process for flagged accounts. If available, use this process to explain your situation and request a review of the flagging.

6. **Secure Your Account:**
   - Ensure that your account is secure by changing your password and enabling two-factor authentication if the platform offers this feature. This can help prevent unauthorized access.

7. **Be Patient:**
   - Depending on the platform, it may take some time to resolve the flagging issue. Be patient and continue to follow the platform's guidelines while waiting for a response from support.

8. **Learn and Improve:**
   - Use this experience as an opportunity to learn about the platform's rules and policies. Make an effort to understand how to use the platform in compliance with its guidelines moving forward.

Remember that each platform may have its own specific procedures for addressing flagged accounts, so it's essential to refer to their guidelines and seek assistance directly from their support channels.

If you're unsure about the reason for the flagging or need specific guidance related to a particular platform, it's best to contact their support team for assistance. They can provide you with the most accurate information and help you resolve the issue effectively.

**Reply by paulospx** [reply]
Until now I have not received any substantive communication from GitHub for over two months.

This lack of response and transparency raises serious concerns about the reliability and integrity of GitHub's security and flagging systems. It appears these systems may be compromised, as they have not functioned properly in handling my case. This situation is not only frustrating but also significantly disrupts my ability to conduct my normal work activities.

I am disappointed with the unprofessional handling of my case by GitHub, particularly given the lack of support and communication. It is crucial for platforms like GitHub to maintain a responsive and efficient support system to uphold user trust and ensure the security of their accounts.

Best regards,
Paulo

**Comment by TICBVN** [comment]
Please I have the same problem. I've been waiting 29 days and no update.

**Reply by mhamidjamilraza** [reply]
same, more then a month

**Comment by leox01code** [comment]
Hi, my organization has been flagged. Please help me.

**Comment by naftalynk** [comment]
My account has been flagged. I didn't break any rules

**Comment by vaskevich-roman** [comment]
Hello! My account was marked, and I didn't break any rules.

**Comment by MelanieStehr** [comment]
I have the same problem and would be greatful for any support.. Thanks in advance.

**Comment by WenqingWang-07** [comment]
Hi, my account has been flagged too, please remove the mark!

**Comment by undoPlease** [comment]
I have been flagged to... just return to git after a break.

**Comment by drobotalex1** [comment]
Hi! 
My account has also been flagged, and I didn't break any rules.

**Comment by m0riii** [comment]
my account has been flagged.
please help me

**Comment by paulvamato** [comment]
My account has been flagged, it's a new account for a new company that I work for
Thanks in advance.

**Comment by sidigs-id** [comment]
Hi, I'm having the same problem now

**Comment by AsmBl4de** [comment]
My account has also been flagged, and I didn't break any rules.

Can you please raise this topic for the help guys?

**Comment by heomi0928** [comment]
I have the same problem and would be greatful for any support.. Thanks in advance.

**Comment by fu3001** [comment]
My account has been flagged and I didnt break any rules.

Please unflag my account again. Thanks in advance.

**Comment by kakarot3lv21** [comment]
My account has been flagged. I didn't break any rules

**Reply by Cystemzz** [reply]
Did it get resolved?

**Comment by Antitrack** [comment]
Immidately after creating my account I was informed it was flagged.
What nonsense is going on here ? I have not written anything !

**Comment by linas-p** [comment]
Department [VU-MIF-SE](https://github.com/VU-MIF-SE) organization has been flagged.

**Comment by hindustaantech** [comment]
Hi,

I just created my organization account, but it's got flagged.
I need help in making this account public.

Regards,
Shubham Bajaj, CEO & CTO
Hindustaan Tech

**Comment by angelabelton** [comment]
Hi There,

My account is flagged, please help to fix it.

Thanks.

**Comment by JurianvD** [comment]
Same problem for me. Created an account three weeks ago, got flagged immediately. I need it for work.
No response from support on my tickets for three weeks.
Anyone with a suggestion what I might do? Or why a new account gets flagged?

**Comment by bnd-api-pr-approver** [comment]
Hi, My account has been flagged (incorrectly) please can this be looked at? Thank you

**Comment by YossefMohamed** [comment]
My account has been flagged. I didn't break any rules
any help?

**Comment by galvadiz** [comment]
Hi, my account has been flagged. I need to reinstatement my account soon. I can delete whatever it takes to get it back and never do anything against the normes again. Please, I need this account to work.

**Comment by superman285** [comment]
My account has also been flagged, and I didn't break any rules.
I created tickets but i didn't get any reply for many days.
Can you please raise this topic for the help guys?

**Comment by statukocu** [comment]
My account has been flagged and I didnt break any rules.

**Comment by Bastian-Z3ro** [comment]
tengo el mismo problema y no sé como contactar al soporte.

**Comment by pauldropz** [comment]
my account is flagged for more than a week now......................please raise a topic

**Comment by drewsmith6** [comment]
My account has been flagged for a few days and need it available to the public for an assignment. Can you please help me get this checked out

**Comment by yoblogger** [comment]
My account has been flagged. I didn't break any rules
 my account is new .

**Comment by mdahteshamulhaque** [comment]
My account has been flagged and I didnt break any rules.

Please unflag my account again. Thanks in advance.

**Comment by Lucasciampi** [comment]
My account has been flagged, and I didn't break any rules. I've already contact support to solve my problem, but I've not received any answer. Someone, please could help me to solve this problem?

**Comment by Pizzarrone1** [comment]
Flagged as spam
I didn't break any rule, I open a ticket but no one ever answered since January.

**Comment by liuyiqun2022** [comment]
My account has been flagged,Please support,thanks.

**Comment by mahmoodhamza1261** [comment]
my account has been flagged, i did not violate any rules, how can my account can be unflagged

**Reply by admzpr8** [reply]
my account has been flagged, i did not violate any rules, how can my account can be unflagged ?

**Comment by pnagarsekar** [comment]
My Account is flagged as well and no one in team can access my urls.

**Comment by user5060** [comment]
my account has been flagged, i did not violate any rules, please unflagged/restore my account

**Comment by Case-RC** [comment]
My account has been flagged, I would like to use my account to work with others.

**Comment by MrProcastinator** [comment]
How do I contact support?
My account has been flagged at account creation, I made a ticket but I had no response in 20 days.

**Reply by GIAN-55** [reply]
Did you solve it? How?

**Comment by g-dsnk9000** [comment]
What happend to my account? Suddenly I got flagged and marked as spam??! I did nothing wth.

**Comment by wkofficials** [comment]
My account was marked, and I didn't break any rules.

**Comment by ucoursein** [comment]
My account has been flagged. hope I didn't break any rules by mistake.

**Comment by Elooh** [comment]
My account has been flagged and I didn't break any rules. I am trying to contact support

**Comment by mdwasim2** [comment]
My account has also been flagged, and I didn't break any rules.

Can you please raise this topic for the help guys?

**Comment by gaurab0** [comment]
My account has been flagged.
How does that happen?

**Comment by capellavacancies** [comment]
same here, got flagged I changed my email but nothing changed

edit:
I can't contact support because when I create a new ticket I get:
_We've received a few new tickets from your account recently. If you'd like to add additional information you can add a comment to an existing ticket, or wait a few minutes before opening a new ticket._

I opened just one ticket and waited for days before trying to open another one

**Reply by devtmj1123** [reply]
same here

**Comment by amirminaeian** [comment]
MY account has been flagged. I'm new

**Comment by Zonloi** [comment]
My account has also been flagged, and I didn't break any rules. Please help me. I cannot authorize a third party application.

**Comment by Bellotti228** [comment]
My account was also flagged as spam, but I didn't violate anything, help me solve this problem

**Comment by Erik-Egnatosyan** [comment]
My account has also been flagged, and I didn't break any rules.

**Comment by D3troit98** [comment]
My account has been flagged too. For over a week now, with no reply

**Comment by kangmiru** [comment]
My account has also been flagged, and I didn't break any rules.

**Comment by gooddavvy** [comment]
My account has been flagged with 1 reply. I didn't break any rules, too.

I got unflagged today!

**Comment by Dmambo** [comment]
Great to hear!

**Reply by gooddavvy** [reply]
:D

Have you?

**Comment by ADHULE** [comment]
hello the community am a beginner in programming, because am a student by this occasion I would like to ask you how to communicate the arduino card with the eclipse? please help me

**Reply by gooddavvy** [reply]
You open that in a new discussion, not an unrelated discussion. Therefore everyone will ignore this comment.

**Comment by ADHULE** [comment]
hello the community am a beginner in programming, because am a student by this occasion I would like to ask you how to communicate the arduino card with the eclipse? please help me

**Reply by gooddavvy** [reply]
You open that in a new discussion, not an unrelated discussion. Therefore everyone will ignore this comment.

**Comment by AbdulazizDXT** [comment]
I'm also unfortunately flagged

**Comment by KiaQuest** [comment]
Github says me to come here for solution (my acc get marked too) so now where is the solution ? 😂😂😂😂😂😂😂😂
all ppl here have a problem just as like as me 😂😂😂
and github support email not work 😂😂

what a joke system ...

**Reply by paulospx** [reply]
I have the same issue here.
extremely frustrating.

**Comment by hum4noidx** [comment]
The same thing happened to me, I sent an appeal and a request to the support service, but there is no response yet

**Reply by AbdulazizDXT** [reply]
The solution is to beg them, unfortunately, if your account is important, and also you should not have violated anything, or if you violated, I apologize to them and tell them I did not mean it, and I am sorry, and there is no one to help me except you, and most likely the mark will be removed

**Reply by hum4noidx** [reply]
In order to beg them, I need to at least get some kind of answer. I was flagged for trade control sanctions violation, but the joke is that I did not leave my country and did not visit these regions...

I received such a letter:


_It appears your account may be based in a U.S.-sanctioned region. As a result, we are unable to provide private repository services and paid services for your account. GitHub has preserved, however, your access to [certain free services for public repositories](https://docs.github.com/en/github/site-policy/github-and-trade-controls#what-is-available-and-not-available). If your account has been flagged in error, and you are not located in or resident in a sanctioned region, please [file an appeal](https://docs.github.com/en/github/site-policy/github-and-trade-controls#how-is-github-ensuring-that-folks-not-living-in-andor-having-professional-links-to-the-sanctioned-countries-and-territories-still-have-access-or-ability-to-appeal). Please read about [GitHub and Trade Controls](https://docs.github.com/en/github/site-policy/github-and-trade-controls) for more information._

**Reply by chomeso3o** [reply]
You get tickets and waiting an hour ,the problem will be solved.i just like that.look picture
![CBFACAC2-72DC-4C35-BBD2-651A63B96DD3](https://github.com/community/community/assets/121551553/cd743857-2df5-4cdf-81d0-876d93ca5f27)

**Comment by kunmifab** [comment]
I got this message when i changed my password and tried to connect my GitHub account to my code editor "You are marked as spam, and therefore cannot authorize a third party application.". What could be the reason and how do i get it unflagged. My team members are not seeing my PR also.

**Comment by KiaQuest** [comment]
Thank  You Very much , my problem has solved <3

**Comment by K-O-V-I-D** [comment]
@draganali  I have created two support tickets, it's has been 15+ days with no response from the support team, actually, my account was flagged and I don't even know the reason!! Can anyone let me know where  I can get a sooner reply as my campus placements are starting..please help!!

**Comment by anuraganalyst** [comment]
Hi Guys!! Suddenly my account has been flagged. I didn't break any rules. Can somebody help ?

**Comment by bashangbasir** [comment]
Hi guys, I suddenly got flagged and did not break any rules. Already raise support ticket 2 week ago but didn't hear anything yet. Can somebody help?

**Reply by jackSeigerman** [reply]
was this solved for you yet? I am in the same boat rn and its also been two weeks.

**Comment by DamiaNeoDon** [comment]
hi, please consider contacting support, your account must be a new one

**Reply by mhamidjamilraza** [reply]
I have contacted, my account was not new, it was created in 2019, and has more then 900 commits.

**Comment by DamiaNeoDon** [comment]
hi, please consider contacting support, your account must be a new one

**Reply by mhamidjamilraza** [reply]
I have contacted, my account was not new, it was created in 2019, and has more then 900 commits.
how bad they are at resolving this issue, i had send them mail before month from the main account, but they don't response i send mail again after 1 week no response, after that i send mail from this account not linked with the flagged account they reply to this mail that i should send mail from the Gmail account which was linked with the flagged account, I mean what the hell is this behavior, @github is not responding on associated mail (for approx 1 month and till now) and replayed on other mail in a day.

**Comment by paulospx** [comment]
Looking at the numerous users, including myself, which suggests potential issues with GitHub's flagging system. This could indicate either a malfunction within the system or, more concerning, misuse of the flagging process.

Given the critical nature of this issue, I expect a higher level of professionalism and promptness from GitHub in handling such cases. It is vital for GitHub to address these concerns swiftly to maintain user trust and ensure the integrity of its platform.

Regards,
Paulo

**Comment by cuikeyao** [comment]
Who can help me? My account has also been flagged, and I have submitted a ticket, but there has been no response for two weeks. What should I do?

**Comment by paulospx** [comment]
How can this be marked as 'answered', if the support team themselves are unreachable and unresponsive, that's a whole different story - a complete and utter failure of the system. You can't just mark something as 'answered' and then ignore the people who are still struggling with the issue, that's not how you build trust or provide good customer service.

**Comment by HimanshuMohanty-Git24** [comment]
I Have been flagged and there is no response from GitHub Support till now.

**Comment by jackSeigerman** [comment]
same, its been two weeks since I sent my reinstatement request. I was not warned upon getting flagged. My email is my own, and I never spammed or did anything wrong.  I have heard nothing back yet.

**Comment by khayym** [comment]
Hi please help me. My account flaged 

 #2895652

**Comment by alifyudha** [comment]
please if any github support read this, read my ticket please, i am getting flagged for a reason i dont know why,. i can't even login to 3rd party apps MarsCode because iam getting flagged, there is important project here that i didnt not backup it yet, please. i beg you github support

**Comment by JoshuaDevCo** [comment]
i hav e been flagged for 3 months and nobody is helping

**Reply by reckDevice** [reply]
Do you have found a way to fix this problem?

**Comment by omar-m-z-dotcom** [comment]
Hello, I'am currently facing some issues with deleting the organization "Omar-Business-intelligence-repos" which I planned to use for grouping my personal github repos (not for an actual organization).
Little did I know that this was a very bad idea and there were smarter ways to do grouping, hence I tried to delete this organization but wasn't able to do it. 
Because my account has been flagged by mistake I tried to contact customer support about this but I couldn't because my subscription on github is free and I believe this issue is interfering with my use of github copilot on vscode.
I would appreciate it if I can get a response about this issue as soon as possible.

also here is some footage and some logs that could help in solving the problem:
![WhatsApp Image 2024-11-15 at 02 56 04_a32dedd4](https://github.com/user-attachments/assets/006e7dea-7d94-48fa-88b8-5d8c2a5fe4fe)
![WhatsApp Image 2024-11-15 at 02 31 59_8c8e603e](https://github.com/user-attachments/assets/7cafb0c8-67cd-41c9-8d3b-da448433c144)
![WhatsApp Image 2024-11-15 at 02 29 54_106f5aa8](https://github.com/user-attachments/assets/6f551459-e3b8-4c35-99c5-f17d45d7158d)

1. output on vscode from Developer: GitHub Copilot chat Diagnostics:
## GitHub Copilot Chat

- Extension Version: 0.23.2024102903 (prod)
- VS Code: vscode/1.95.3
- OS: Windows

## Network

User Settings:
json
  "github.copilot.advanced": {
    "debug.useElectronFetcher": true,
    "debug.useNodeFetcher": false
  }


Connecting to https://api.github.com/:
- DNS ipv4 Lookup: 140.82.121.6 (6 ms)
- DNS ipv6 Lookup: Error: getaddrinfo ENOTFOUND [api.github.com](http://api.github.com/)
- Electron Fetcher (configured): HTTP 200 (213 ms)
- Node Fetcher: HTTP 200 (203 ms)
- Helix Fetcher: HTTP 200 (214 ms)

Connecting to https://api.githubcopilot.com/_ping:
- DNS ipv4 Lookup: 140.82.113.22 (8 ms)
- DNS ipv6 Lookup: Error: getaddrinfo ENOTFOUND [api.githubcopilot.com](http://api.githubcopilot.com/)
- Electron Fetcher (configured): HTTP 200 (897 ms)
- Node Fetcher: HTTP 200 (1425 ms)
- Helix Fetcher: HTTP 200 (421 ms)

## Documentation

In corporate networks: [Troubleshooting firewall settings for GitHub Copilot](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot).

2. output on vscode from github copilot logs:
2024-11-15 02:53:31.756 [info] Logged in as omar-m-z-dotcom
2024-11-15 02:53:32.066 [warning] Invalid copilot token: missing token: 403
2024-11-15 02:53:32.066 [info] copilot token chat_enabled: undefined, sku:
2024-11-15 02:53:32.076 [info] Logged in as omar-m-z-dotcom
2024-11-15 02:53:32.360 [warning] Invalid copilot token: missing token: 403
2024-11-15 02:53:32.360 [error] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support. You are currently logged in as omar-m-z-dotcom."
2024-11-15 02:53:32.360 [warning] [LanguageModelAccess] LanguageModel/Embeddings are not available without auth token
2024-11-15 02:53:32.360 [error] Error: Contact Support. You are currently logged in as omar-m-z-dotcom.
    at tC._authShowWarnings (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:3282)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at tC.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:2135)
    at UR.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:282:39062)
    at gC._getAuthSession (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2552)
    at t (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2015)
2024-11-15 02:53:32.361 [warning] [LanguageModelAccess] LanguageModel/Embeddings are not available without auth token
2024-11-15 02:53:32.361 [error] Error: Contact Support. You are currently logged in as omar-m-z-dotcom.
    at tC._authShowWarnings (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:3282)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at tC.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:2135)
    at UR.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:282:39062)
    at gC._getAuthSession (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2552)
    at t (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:631)

3. name: omar mohamed
4. username: omar-m-z-dotcom
5. email: triro765@gmail.com
6. organization to be deleted: Omar-Business-intelligence-repos

thank you.

**Comment by reckDevice** [comment]
>please if any github support read this, read my ticket please, i am getting flagged for a reason i dont know why,. i can't even login to 3rd party >apps MarsCode because iam getting flagged, there is important project here that i didnt not backup it yet, please. i beg you github support
I met the same problem. someone can help?

**Comment by Mike6088** [comment]
Mine has been flagged  as spam as well, newly created about a week ago

**Comment by fayaznasrati** [comment]
Mine has been flagged too, its been two weeks not response yet from GitHub support

**Comment by jaydipkanpariya** [comment]
How to remove this flag?????????
Support, not give me a response
I`m not able to add any collobratore in my repo

**Comment by telemac** [comment]
i am flagged too, really don't know why, I can't see a reason. I am a freelance developer sharing repos with customers, the situation is really problematic for me. I have an open ticket to the support, but no answer. When I read the delays some of you are waiting without answers I am quite concerned.

**Comment by jobrien12345** [comment]
It was not suspended.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
